api/router.go

```diff
@@ -165,7 +165,7 @@ func NewRouter(s *Server) *mux.Router {
 	r.Path("/v1/ws/settings").Handler(f(schemas, settingListStream))
 	r.Path("/v1/ws/{period}/settings").Handler(f(schemas, settingListStream))
 
-	volumeListStream := NewStreamHandlerFunc("volumes", s.wsc.NewWatcher("volume", "engine", "replica"), s.volumeList)
+	volumeListStream := NewStreamHandlerFunc("volumes", s.wsc.NewWatcher("volume", "engine", "replica", "backup"), s.volumeList)
```

Comment:
Do we need anything from UI side to adapt to this change?

Reply:
No, UI focuses on the RESTful/WebSocket endpoint response.
The longhorn manager backend handles to get the `backupstatus` from Backup CR now (originally is from Engine CR).